src/analyzer/protocol/snmp/SNMP.cc

```diff
@@ -2,10 +2,7 @@
 
 #include "zeek/analyzer/protocol/snmp/SNMP.h"
 
-#include "zeek/Func.h"
 #include "zeek/Reporter.h"
```

Comment:
Just because it stands out: Why is Reporter.h needed here?

Reply:
That's a darn good question. It gets included by the `snmp_pac.h` via `SNMP.h` too. There's so much about what gets included by the pac files it's crazy.